File: jplister.java
#!/usr/bin/env java
import java.util.List;
import org.jspecify.Nullable;
import org.jspecify.NonNull;

public @Nullable T first( @NonNull List<T> container ) {
   return container.isEmpty() ? null : container.get(0);
}

void main() {
  var listOne = List.of("a", "b", "c");
  var listTwo = List.of(1, 2, 3);
      System.out.println(listOne.getFirst());
    System.out.println(listTwo.getFirst());
}

File: jplister.py
#!/usr/bin/env python
# Notice my useage of a tuple and not a list, which is already immutable in python
def main():
    listOne = ( 'a', 'b', 'c' )
    listTwo = ( 1, 2, 3 )
    print(listOne[0])
    print(listTwo[0])

if __name__ == "__main__":
    main()
